wurst/_handles/Lightning.wurst

```diff
@@ -0,0 +1,30 @@
+package Lightning
+import NoWurst
+import Vectors
+import Colors
+
+public function addLightning(string codeName, bool checkVisibility, vec2 start, vec2 finish) returns lightning
```

Comment:
"finish" should probably be "end" on these functions.
Also what about adding all the default lightning strings in this package?

Reply:
I had trouble that "end" is a keyword so I can't use that name.
Well, ok, I'll do it if you mind not something like "_end"